modules/operator-marketplace.adoc

```diff
@@ -1,16 +1,57 @@
 // Module included in the following assemblies:
 //
 // * operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
 
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
 [id="marketplace-operator_{context}"]
-= Marketplace Operator
+ifdef::operator-ref[= Marketplace Operator]
+ifdef::cluster-caps[= Marketplace capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Marketplace Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
 
 [discrete]
 == Purpose
 
-The Marketplace Operator is a conduit to bring off-cluster Operators to your cluster.
+ifdef::cluster-caps[]
+
+The Marketplace Operator provides the features for the `marketplace` capability.
 
+endif::cluster-caps[]
+
+The Marketplace Operator simplifies the process for bringing off-cluster Operators to your cluster by using a set of default Operator Lifecycle Manager (OLM) catalogs on the cluster. 
+
+If you disable this capability, the catalogs can still be configured and managed on the cluster manually. If enabled, the individual catalogs can be enabled and disabled by configuring the Marketplace Operator.
```

Comment:
Did you want to wrap this in `ifdef::cluster-caps[]`?  Possibly with:

> Individual catalogs can be enabled and disabled by configuring the Marketplace Operator.

or some such in an else case, to tell those folks that there are still per-catalog knobs, even if they can't disable the whole component?

Reply:
Thank you for the feedback! I have updated the content with your suggestion.